src/backend/linux_raw/vdso.rs

```diff
@@ -440,7 +440,7 @@ fn test_vdso() {
         let ptr = vdso.sym(cstr!("LINUX_2.6"), cstr!("__vdso_clock_gettime64"));
         #[cfg(target_arch = "riscv64")]
         let ptr = vdso.sym(cstr!("LINUX_4.15"), cstr!("__vdso_clock_gettime"));
-        #[cfg(target_arch = "powerpc64")]
+        #[cfg(any(target_arch = "powerpc", target_arch = "powerpc64"))]
         let ptr = vdso.sym(cstr!("LINUX_2.6.15"), cstr!("__kernel_clock_gettime"));
```

Comment:
What would you think about using `__kernel_clock_gettime64` for powerpc?

According to [the Linux docs](https://man7.org/linux/man-pages/man7/vdso.7.html), ppc/32 has a `__kernel_clock_gettime64` defined, as of Linux 5.11. Rust [supports](https://doc.rust-lang.org/nightly/rustc/platform-support.html) Linux  3.2 on powerpc. So, `__kernel_clock_gettime64` may not be available, but if it isn't, we can just skip storing anything in `CLOCK_GETTIME` here, which will leave it set to `rustix_clock_gettime_via_syscall`, which is slower but works, and it'd only be used on Linux < 5.11.

That would avoid the need for `ClockGettimeTimeType`.

Reply:
I've now fixed the flate2 issue on main in https://github.com/bytecodealliance/rustix/pull/1353.

Reply:
Excellent. Thanks. I'll rebase later on and hopefully that should sort everything.

Reply:
Also, did you see this guy?
https://github.com/sunfishcode/linux-raw-sys/issues/148

Reply:
I've now fixed the 32-bit ioctl values in linux-raw-sys 0.9.0; please update to that in this PR!

Reply:
Also, it looks like the updated code here uses "__kernel_clock_gettime" for ppc; it should be using "__kernel_clock_gettime64", right?

Reply:
I think the github UI isn't showing things very clearly. The second commit in the PR [here](https://github.com/bytecodealliance/rustix/pull/1337/commits/bf14d181b770c8aacd4af24474ab3766ffc57591), changes things to use `__kernel_clock_gettime64 `

Reply:
I've bumped `linux-raw-sys` and re-instated the failing `ioctl` test.

Reply:
It looks like the symbol in vdso_wrappers.rs is updated, but the `test_vdso` test in vdso.rs isn't.

Reply:
(We should ideally tidy up the redundancy of having the symbol name appear in two places like this, but for now, that's what we have.)

Reply:
> It looks like the symbol in vdso_wrappers.rs is updated, but the `test_vdso` test in vdso.rs isn't.

My bad. Sorry, didn't realise this cropped up twice. I've just pushed the fix.

Reply:
Just had to add [this](https://github.com/bytecodealliance/rustix/pull/1337/commits/66593e4ebb52917b29cfa895550c64adc271a56a). It seems that the runner on the CI may not have a new enough kernel? Obviously it doesn't matter what OS the job is running in since the container will still be running on the hosts kernel no matter what.

Reply:
> Ok. I think this should be good to merge now if you're happy with it? Will you make a new dot release soon also so I can pick it up on crates.io?

It happens that rustix is in the process of preparing for a major semver bump. I've now published [1.0.0-prelrease.1](https://crates.io/crates/rustix/1.0.0-prerelease.1) which contains this patch. I hope to publish a final 1.0.0 if no problems are found in testing.

Reply:
Great. Thanks.